topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -0,0 +1,743 @@
+---
+layout: tutorial_hands_on
+
+title: Identification of allelic variants in SARS-CoV-2 from deep sequencing reads
+level: Introductory
+zenodo_link: "https://zenodo.org/record/5036687"
+questions:
+- How can we extract annotated allelic variants in SARS-Cov-2 sequences in Galaxy?
+- Which tools can we use to identify SARS-CoV-2 clades in Galaxy?
+objectives:
+- Repeat SARS-CoV-2 data preparation
+- Select and execute workflow to extract annotated allelic variants from FASTQ files
+- Execute workflow to summarize annotated allelic variants
```

Comment:
Run workflow to summarize and generat report for previously called allelic variants